faqs/galaxy/workflows_import_from_dockstore.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Import workflows from DockStore
+area: workflows
+box_type: tip
+layout: faq
+contributors: [nekrut]
+---
+
+[Dockstore](https://dockstore.org/) is a free and open source platform for sharing reusable and scalable analytical tools and workflows.
+
+-----
+
+The following short video walks you through this uncomplicated procedure:
+
+<p align="center"><iframe width="560" height="315" src="https://www.youtube.com/embed/K2wFrSLFpa0?si=mcaCXKNx3JNbLnBv" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe></p>
```

Comment:
this is, unfortunately, an inaccessible tip box. It's fine if you want to include the video, but at minimum we **need** a textual description of the process (with relevant icons).

Reply:
why is it inaccessible?

Reply:
close your eyes. Click on the youtube video. Can you follow the steps presented in that video? no, right? There's no audio description (which is good! it would take longer to update those videos, that's fine!), so a visually impaired user is stuck here.

If they were text steps written out in this tip box, then they'd be read out to someone using a screenreader. Thus the video would be an optional addition for someone who prefers that format, but we wouldn't be making vision a mandatory part of accessing the materials.

Reply:
the video can stay, just make sure the same information is also available as text is all (not only more accessible, but also easier to update when the Galaxy interface changes later)